conf/messages

```diff
@@ -163,11 +163,38 @@ departure.details.error.overdue = Date and time of departure cannot be more than
 departure.details.error.future = Date and time of departure cannot be in the future
 
 location.title = Location
-location.question = What is the code for the location of the goods when they are declared?
-location.hint = This is usually the port they are leaving from and is officially known as the Goods Location Code. For example, if they were at Felixstowe, it would be GBAUFXTFXTFXT.
+location.question = What is the goods location code?
+location.hint.paragraph1 = This is a code for the location where goods are presented to customs. If necessary the goods location code must be updated as part of the arrival process, when the goods arrive at that location.
+location.hint.paragraph2 = Most goods location codes contain just letters. For example, for goods leaving from Felixstowe, it would be GBAUFXTFXTFXT.
 location.code.empty = Enter a goods location code
 location.code.error = Enter a goods location code in the correct format
 
+location.expander.title=Find the goods location code (CDS data element 5/23)
+location.expander.intro=Select the location type, and then look up the code (links open in a new tab).
+location.expander.paragraph1.title=RoRo ports for road transport
+location.expander.paragraph1.text={0}<br/>Download the Tariff supplement and search for appendix 16R.
```

Comment:
There are still a couple lines where you're using `<br/>` tag. Is it possible to get rid of that the same way as you did for others?

Reply:
Ok, I've implemented the same sort of fix for the `<br>` tags but personally I'm not a fan. Not sure what the benefit of not having the simple tags like `<p>` and `<br>` in the messages file is, but IMO the resulting verbose code is much harder to reason about (as opposed to having these simple tags in the messages file).